cloud/common/deploy/provider/pulumi.go

```diff
@@ -201,10 +203,15 @@ func (s *PulumiProviderServer) Up(req *deploymentspb.DeploymentUpRequest, stream
 		return err
 	}
 
-	_, err = autoStack.Refresh(context.TODO())
-	if err != nil {
-		logger.Errorf(err.Error())
-		return err
+	refresh, ok := attributesMap["refresh"].(bool)
```

Comment:
can also namespace this property if necessary. All pulumi providers will implement this.